PubSub/src/Subscription.php

```diff
@@ -399,11 +408,42 @@ public function create(array $options = [])
         }
 
         $options = $this->formatSubscriptionDurations($options);
+        $options = $this->formatDeadLetterPolicyForApi($options);
 
-        $this->info = $this->connection->createSubscription([
-            'name' => $this->name,
-            'topic' => $this->topicName
-        ] + $this->formatDeadLetterPolicyForApi($options));
+        // convert optional args to protos
+        if (isset($options['expirationPolicy'])) {
+            $options['expirationPolicy'] = $this->requestHandler->getSerializer()->decodeMessage(
+                new ExpirationPolicy(),
+                $options['expirationPolicy']
+            );
+        }
```

Comment:
For V2, we will want to do something like:
```suggestion
        $request = new CreateSubscriptionRequest();
        if (isset($options['expirationPolicy'])) {
            $request->setExpirationPolicy($this->requestHandler->getSerializer()->decodeMessage(
                new ExpirationPolicy(),
                $options['expirationPolicy']
            ));
        }
```